controllers/slicegateway/slicegateway.go

```diff
@@ -397,6 +412,21 @@ func (r *SliceGwReconciler) deploymentForGatewayClient(g *kubeslicev1beta1.Slice
 								}},
 							},
 						},
+						PodAntiAffinity: &corev1.PodAntiAffinity{
```

Comment:
Please write a func that returns corev1.PodAntiAffinity and call that func here. Same func should be called in the server deployment func.